casadm/safeclib/LICENSE&COPYING.txt

```diff
@@ -1,6 +1,6 @@
 Safe C Library
 
-          Copyright (c) 2014-2016, Intel Corporation. All rights reserved.         
+          Copyright (c) 2014-2020, Intel Corporation. All rights reserved.         
```

Comment:
We don't want to update copyright in safeclib. This library is not maintained by us.